docs/pages/linking/into-your-app.mdx

```diff
@@ -9,6 +9,8 @@ import { Terminal } from '~/ui/components/Snippet';
 
 This guide provides steps to configure standard **deep links** in your project by adding a custom scheme.
 
+> **info** For most apps, you probably want to set up [Android App/iOS Universal Links](/linking/overview/#universal-linking) instead of the deeps links documented here, or set up both.
```

Comment:
```suggestion
> **info** For most apps, you probably want to set up [Android App/iOS Universal Links](/linking/overview/#universal-linking) instead of the deep links described in this guide or set up both.
```